pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.go

```diff
@@ -34,12 +34,20 @@ type Conf struct {
 	Tags *[]Tag `json:"tags,omitempty"`
 }
 
-// Only one of zipkin or datadog can be used.
+// Only one of zipkin, datadog or opentelemetry can be used.
 type Backend struct {
 	// Zipkin backend configuration.
 	Zipkin *ZipkinBackend `json:"zipkin,omitempty"`
 	// Datadog backend configuration.
 	Datadog *DatadogBackend `json:"datadog,omitempty"`
+	// OpenTelemetry backend configuration.
+	OpenTelemetry *OpenTelemetryBackend `json:"opentelemetry,omitempty"`
+}
+
+// OpenTelemetry tracing backend configuration.
+type OpenTelemetryBackend struct {
+	// Address of OpenTelemetry collector e.g. otelcol:4317.
```

Comment:
You can try adding the new kubebuilder `example` directive here like

```suggestion
	// Address of OpenTelemetry collector.
	// +kubebuilder:example="otel-collector:4317"
```